docs/docs/examples/guides/ur-harware-interface.md

```diff
@@ -0,0 +1,577 @@
+internal development for robot-specific feature integration. The UR hardware collection provided by AICA comes with
+special tools and functionalities that are unique to UR robots. This guide intends to explain these concepts and how
+they can be leveraged in AICA Studio.
+
+To use the UR collection, add the latest version of `collections/ur-collection` to your configuration in AICA Launcher.
+Doing this will add multiple new hardware examples as well as a few controllers to AICA Studio. All UR robots share the
+same hardware interface in AICA Studio. The hardware interface has quite a large number of parameters, most of which are
+not important for regular use cases.
+
+:::tip
+
+For best results with a UR robot, always set the rate of the hardware interface to 500 Hertz, which corresponds to the
+control rate of the real hardware.
+
+:::
+
+<div class="text--center">
+  <img src={urHwiStudio} alt="UR Hardware Interface in AICA Studio" />
+</div>
+
+<!-- TO ADD, TBD:
+
+- use of the dashboard controller with local mode and the program node to hand back control between AICA and UR
+- use of the UR impedance controller and hand guiding controller to leverage UR Force mode from AICA Studio
+- use of the dashboard server to observe GPIOs, set payload, zero ft sensor etc -->
+
+<!-- # Hardware Interface
+
+TBD: There is already a [page](../../concepts/building-blocks/hardware-interfaces.md) on hardware interfaces. Are these
+two conflicting? Should we link to this one?
+
+Within the context of AICA Studio, but also in ROS, hardware interfaces are used as middleware between controllers and
+actual hardware. In other words, they are tasked with two-way communication of commands and state feedback between the
+robot and the controller that usually lies above. -->
+
+## Local and Remote Control
+
+The concept of Local and Remote Control on PolyScope can be easiest explained by introducing a _primary_ and _secondary_
+device architecture. In Local Control, the controller is the _primary_ and has full authority on loading and starting
+programs. In other words, the robot has to be used in person through the teach pendant and any commands sent from an
+external source will be rejected. On the other hand, Remote Control allows to control the robot via external sources,
+such as sockets, I/Os and the Dashboard Server. In this case, the controller is the _secondary_ and external sources can
+load and start programs or directly send URScript commands to the controller.
+
+:::note
+
+Safety features remain active in Remote Control.
+
+:::
+
+Choosing one of the two modes depends on the specific task at hand. During a development phase, it might be preferable
+to create the programs in Local Mode, whereas in a production setting, PLCs would be responsible to load and start the
+desired programs while the robot is in Remote Control. With the AICA System, users have the possibility to get the best
```

Comment:
```suggestion
desired programs while the robot is in Remote Control. With the AICA System, users have the chance to get the best
```

nit: not the best alternative with my suggestion, but "possibility" somewhat reads weird to me